pkgs/by-name/README.md

```diff
@@ -112,12 +112,11 @@ There's some limitations as to which packages can be defined using this structur
 
 CI performs [certain checks](../test/nixpkgs-check-by-name/README.md#validity-checks) on the `pkgs/by-name` structure.
 This is done using the [`nixpkgs-check-by-name` tool](../test/nixpkgs-check-by-name).
-The version of this tool used is the one that corresponds to the NixOS channel of the PR base branch.
-See [here](../../.github/workflows/check-by-name.yml) for details.
 
-The tool can be run locally using
+You can locally emulate the CI check using
 
-```bash
-nix-build -A tests.nixpkgs-check-by-name
-result/bin/nixpkgs-check-by-name .
 ```
+$ ./pkgs/test/nixpkgs-check-by-name/scripts/run-local.sh master
```

Comment:
Why not put it inside `maintainers/scripts` ?

Reply:
`run-local.sh` is indeed a script only for maintainers, but there's also `fetch-tool.sh` next to it, which is used by both `run-local.sh` and CI, so that wouldn't strictly be a maintainer script.

I guess `run-local.sh` should be in `maintainers/scripts`, while `fetch-tool.sh` could stay in `pkgs/test/nixpkgs-check-by-name/scripts`, though then they're a bit far apart, and the `pkgs/by-name` implementation is already split into like 5 different places.

Reply:
I'm all for keeping things together.
If we nonetheless insist on having `maintainers/scripts`, it could be a minimal wrapper that invokes the real thing.

Reply:
Sounds like a symlink cost nothing here?

Reply:
Yeah that's an idea. I'll do the symlink and make its path be printed in CI.

Reply:
Now introduced `maintainers/scripts/check-by-name.sh` as a symlink in https://github.com/NixOS/nixpkgs/pull/274591/commits/c106c49923fa7cf656ea464b59f74ef623966e82

Reply:
Good to me